### PR TITLE
website main menu: create support menu with community, wiki and commercial submenus

### PR DESCRIPTION
**Pull request by veroandreo**
This PR tries to bring about discussions about re-ordering menus and submenus. It's yet very raw, but the idea is to add a new menu "Support" with 3 children: "Community" (moved from About), "Wiki" (moved from Contribute) and a new "Commercial Support" page. In the latter, the idea would be to list companies or institutions that offer commercial support and services for GRASS (I put mundialis as an example). I used here the same layout as the download index page, so the idea would be to add company/institution logos, plus a description and the link in a button. See below: 

image

As layout for the Support index page, I copied the one from About menu:

image

Content needs to be added/edited and the layouts beautified, but this is a start point to get some feedback, so, please let me know what you think :)

### CODE REVIEW

File: themes/grass/layouts/support/list.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                  <div class="col-11">
+		    <h3 class="mt-2 mb-2"><a href="{{.File.TranslationBaseName}}">{{ .Title }}</a></h3>
+                  </div>
+                </div>
+		</div>
+	       {{ end }}  
+            </div>
+          </div>
+        </div>
+      </section>{{ partial "footer.html" . }}
```
Comment (nobeeakon) [inline themes/grass/layouts/support/list.html:50]: weirdly footer.html has a closing </body></html> which turns this into invalid html, an option would be to move it at the end of the file

Comment (veroandreo) [reply themes/grass/layouts/support/list.html:50]: mhm... I didn't change anything of that, just copied the layouts as they were and made minimal changes

File: themes/grass/layouts/support/list.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+{{ "<!-- navigation -->" | safeHTML }}
+<header class="shadow-bottom position-relative">
+  <div class="container">
+    {{ partial "navigation.html" . }}
+  </div>
+</header>
+{{ "<!-- /navigation -->" | safeHTML }}
+
+
+    <section>
```
Comment (nobeeakon) [inline themes/grass/layouts/support/list.html:13]: do we need this additional `<section>`? think is ok to remove

**Comment by neteler** [comment]
Quick feedback: This looks great and I like a lot the splitting out of "Support" into an own section.

**Comment by neteler** [comment]
The current conflicts need to be solved on command line...

File: themes/grass/layouts/support/commercial.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+        <div class="container">
+
+            <div class="section bg-white mb-4">
+                <div class="row nm">
+                  <div class="col-4 text-center"><object type="image/svg+xml" data="{{.Site.BaseURL}}/images/logos/grasslogo.svg" class="grasslogo"></object></div>
+                  <div class="col-8">
+		    <h3 class="mt-20">mundialis GmbH & Co. KG</h3>
+		    <p>The GRASS GIS source code is available on <a href="https://github.com/OSGeo/grass" target="_blank"><i class="fa fa-github"></i> Github</a>, see all <a href="https://github.com/OSGeo/grass/tags">source code archives</a> for releases (zip and tarball files). Alternatively, find the latest tarballs and snapshots at <a href="https://grass.osgeo.org/grass{{- .Site.Data.grass.current_version_nodots -}}/source/">https://grass.osgeo.org/grass{{- .Site.Data.grass.current_version_nodots -}}/source/</a>.</p>
+            <a class="btn btn-primary" href="https://www.mundialis.de/en/" target="_blank">Visit our website</a>
```
[the remark this answers is not in the document]
Comment (neteler) [reply themes/grass/layouts/support/commercial.html:32]: The logo is available from here: https://data.neteler.org/tmp/mundialis_logo.svg

(I didn't manage to push into this PR)

**Comment by neteler** [comment]
@veroandreo would you mind to fix the current merge conflict on command line?

**Comment by veroandreo** [comment]
> @veroandreo would you mind to fix the current merge conflict on command line?

I only transferred the community page from one folder to another. How do I see these conflicts? 
The command line instructions provided here suggests me to create a new branch, test changes, merge with --no-ff... Are those right? They seem useful for testing someone else's PR

**Comment by veroandreo** [comment]
So, I removed the wiki entry from Support menu:
image

and added an icon to the support panel in home that appears because of this line: https://github.com/OSGeo/grass-website/blob/0bdb7170a1fcf7a12e7302b6261038672ef0bf3c/themes/grass/layouts/index.html#L36
image

Do we want 4 of those?

**Comment by veroandreo** [comment]
fixed logo display
image

**Comment by neteler** [comment]
I am fine with the 4 rather than 3 panels but do not visibly find the Wiki entry any more.

Indeed, it is under the photo on `support/community/` but please let's also (only?) have it in the text.

**Comment by veroandreo** [comment]
> I am fine with the 4 rather than 3 panels but do not visibly find the Wiki entry any more.
> 
> Indeed, it is under the photo on `support/community/` but please let's also (only?) have it in the text.

Indeed. I removed the wiki entry from the support menu as we discussed in the last [PSC meeting](https://trac.osgeo.org/grass/wiki/PSC/Minutes/PSC_Meeting_20230811). 

Do you prefer to keep it as an entry in the support menu instead of the "panel" below the photo in community? IMHO, the Wiki might serve a number of purposes, i.e., contribute, find support, learn... where do you feel it matches better? Well, find support could equate learn too. Currently it is under contribute.

**Comment by petrasovaa** [comment]
> Do you prefer to keep it as an entry in the support menu instead of the "panel" below the photo in community? IMHO, the Wiki might serve a number of purposes, i.e., contribute, find support, learn... where do you feel it matches better? Well, find support could equate learn too. Currently it is under contribute.

This goes into a discussion what the wiki is for. I think we should use it more as a community tool for planning and organizing rather than a place where people can learn or contribute given how easy wiki pages get obsolete. So I would keep wiki in Community, but I agree it deserves more than a picture, probably it could have it's own heading, because it doesn't quite fit into any of those categories on the community page.

**Comment by petrasovaa** [comment]
@veroandreo I added various changes to this PR, it's ready for merge from my side. I changed the layout of the Support page and added short text, added NCSU CGA as commercial support, moved wiki from the panel to the page content within the community page and added fosstodon there. Is web.libera.chat still relevant?

**Comment by neteler** [comment]
Would you mind to share a screenshot here?

**Comment by petrasovaa** [comment]
Selection_269
Selection_270

**Comment by veroandreo** [comment]
> @veroandreo I added various changes to this PR, it's ready for merge from my side. I changed the layout of the Support page and added short text, added NCSU CGA as commercial support, moved wiki from the panel to the page content within the community page and added fosstodon there. Is web.libera.chat still relevant?

Thanks for all the changes @petrasovaa ! I like them! Regarding libera chat, I logged in just out of curiosity and there were 2 others there, but no activity... I'd remove it.

**Comment by veroandreo** [comment]
+1 for merging! :wink:

**Review by wenzeslaus — APPROVED** [review]

**Comment by neteler** [comment]
Not what happened but the "Support" menu is entirely broken:

image


**404**:

- https://grass.osgeo.org/about/community/
- https://grass.osgeo.org/about/community/commercial
- https://grass.osgeo.org/about/community/community

**Pull request merged**